catroid/src/main/res/values-ar/strings.xml

```diff
@@ -326,12 +326,12 @@
   <!-- Project activity view -->
   <string name="project_name">البرنامج:</string>
   <string name="my_projects_activity_title">البرامج</string>
-  <string name="number_of_objects">Objects:</string>
+  <string name="number_of_objects">Objects: %1$d</string>
   <string name="start_scene_name">%1$s (First Scene)</string>
-  <string name="number_of_scripts">النص:</string>
-  <string name="number_of_bricks">اللّبنات:</string>
-  <string name="number_of_looks">المشهد:</string>
-  <string name="number_of_sounds">الأصوات:</string>
+  <string name="number_of_scripts">النص: %1$d</string>
+  <string name="number_of_bricks">اللّبنات: %1$d</string>
+  <string name="number_of_looks">المشهد: %1$d</string>
+  <string name="number_of_sounds">الأصوات: %1$d</string>
```

Comment:
I don't know why but at my local files it all looks okay and in the right order. 
<img width="943" alt="capture1" src="https://cloud.githubusercontent.com/assets/15776520/20265156/b8948f8a-aa6f-11e6-98e6-214c337b107a.png">